packages/auth/src/shared/baseModel.ts

```diff
@@ -1,7 +1,105 @@
-import { Model } from 'objection'
+import {
```

Comment:
:thinking:  What if we moved `baseModel.*` and `pagination.ts` (and `resolvers/page.*`?) to a new (`objection-relay`?) package in the monorepo used by `auth` and `backend`?

We might even be able to define `Model` and `PageInfo` there in a schema that's imported
https://the-guild.dev/graphql/tools/docs/schema-loading